dataframe-jdbc/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/h2/h2Test.kt

```diff
@@ -967,4 +972,127 @@ class JdbcTest {
         }
         exception.message shouldBe "H2 database could not be specified with H2 dialect!"
     }
+
+    // helper object created for API testing purposes
+    object CustomDB: H2(MySql)
+
+    @Test
+    fun `read from table from custom database`() {
+        val tableName = "Customer"
```

Comment:
Most of assertions test correct reading of data, but actually functionality under test is correct propagation of DbType, as i understand. So most of it is duplicated code? If so, changes that will trigger H2 tests assertions to fail, will fail assertions here too. For this reason please consider a different approach here